frame/support/src/traits/misc.rs

```diff
@@ -27,9 +27,22 @@ use sp_std::{cmp::Ordering, prelude::*};
 const DEFENSIVE_OP_PUBLIC_ERROR: &'static str = "a defensive failure has been triggered; please report the block number at https://github.com/paritytech/substrate/issues";
 const DEFENSIVE_OP_INTERNAL_ERROR: &'static str = "Defensive failure has been triggered!";
 
+/// Generic function to mark an execution path as ONLY defensive.
+///
+/// Similar to mark a match arm or `if/else` branch as `unreachable!`.
```

Comment:
From the docs I don't really get what the function is doing.

Reply:
this should be removed.